docs/standard/data/sqlite/connection-strings.md

```diff
@@ -13,7 +13,7 @@ keywords and values.
 
 The following connection string keywords can be used with Microsoft.Data.Sqlite:
 
-### Data source
+### Data Source
```

Comment:
```suggestion
### Data source
```

Per our style guides, these were correct as is. Titles and heading should be in [sentence case](https://docs.microsoft.com/en-us/contribute/style-quick-start#other-style-and-voice-issues-to-watch-for). However, if these are proper names that can be capitalized, but I do not believe that these are.

Reply:
But... they keywords are case-sensitive. I know this goes against casing guidelines, but I think it's important preserve the actual case.

Reply:
Ah, good to know. I was unaware of that.